config/s-nt.h

```diff
@@ -17,7 +17,7 @@
 
 #define OCAML_OS_TYPE "Win32"
 
-#ifdef __MINGW32__
+#if defined(__MINGW32__) || (defined(_MSC_VER) && _MSC_VER >= 1600)
```

Comment:
Is this change related to this PR?

Reply:
The idea is : either we use stdint.h, or we do the typedefs ourselves.